.github/workflows/build-solution.yml

```diff
@@ -89,13 +82,13 @@ jobs:
         run: |
           ./unit-test-solution.sh
 
-      - name: Build application tests container
-        run: |
-          ./build-app-tests-container.sh
+#      - name: Build application tests container
```

Comment:
Add the todo comment to enable this step along with the jira ticket number that involves the todo task to be done.